core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -717,7 +717,7 @@ static ItemExpr *intersectColumns(const RETDesc &leftTable,
     ItemExpr *leftExpr  = leftTable.getValueId(i).getItemExpr();
     ItemExpr *rightExpr = rightTable.getValueId(i).getItemExpr();
     BiRelat *compare = new (bindWA->wHeap())
-      BiRelat(ITM_EQUAL, leftExpr, rightExpr);
+      BiRelat(ITM_EQUAL, leftExpr, rightExpr, TRUE);
```

Comment:
the showplan output is too long. not sure if you can see it correctly... Here is simpler version of explain, it is a nest loop join. 

>>explain options 'f' select eno, ename, dno from t021
 intersect
+>+>select eno, ename, dno from t022;

LC   RC   OP   OPERATOR              OPT       DESCRIPTION           CARD
---- ---- ---- --------------------  --------  --------------------  ---------

4    .    5    root                                                  2.00E+000
2    3    4    nested_join                                           2.00E+000
.    .    3    trafodion_vsbb_scan             T021                  1.00E+000
1    .    2    hash_groupby                                          2.00E+000
.    .    1    trafodion_scan                  T022                  2.50E+001